src/serializers/ng-snapshot.ts

```diff
@@ -0,0 +1,103 @@
+import type { Colors } from 'pretty-format';
+
+// jest hasn't exported this type yet so temporary we copy from 'pretty-format'
+interface PluginOptions {
+  edgeSpacing: string;
+  min: boolean;
+  spacing: string;
+}
+declare type Indent = (arg0: string) => string;
+declare type Print = (arg0: unknown) => string;
+
+const printAttributes = (
+  val: any, // eslint-disable-line @typescript-eslint/no-explicit-any
+  attributes: string[],
+  _print: Print,
+  indent: Indent,
+  colors: Colors,
+  opts: PluginOptions,
+): string => {
+  return attributes
+    .sort()
+    .map((attribute) => {
+      return (
+        opts.spacing +
+        indent(colors.prop.open + attribute + colors.prop.close + '=') +
```

Comment:
```suggestion
        indent(`${colors.prop.open}${attribute}${colors.prop.close}=`) +
```
I'm a fan of backslash-interpolation

Reply:
done